buildSrc/src/main/kotlin/GenerateCompletionScriptsTask.kt

```diff
@@ -311,40 +391,39 @@ abstract class GenerateCompletionScriptsTask : DefaultTask() {
 
         fun getPossibleValues(option: BuildOption<*>, longOptionName: String?): List<String> {
             longOptionName?.let { optName ->
-                HARDCODED_POSSIBLE_VALUES[optName]?.let { return it }
+                HARDCODED_POSSIBLE_VALUES[optName]?.let {
+                    return it
+                }
             }
             return getPossibleValuesList(option)
         }
```

Comment:
💅 This can be simplified:

```kotlin
        fun getPossibleValues(option: BuildOption<*>, longOptionName: String?): List<String> =
            longOptionName?.let { HARDCODED_POSSIBLE_VALUES[it] } ?: getPossibleValuesList(option)
```